schemas/1.22.0

```diff
@@ -0,0 +1,156 @@
+file_format: 1.1.0
```

Comment:
This should be in the [schema-next file](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) NOT in the schemas directory.

Reply:
thanks - might make sense to add that to CONTRIBUTING.md

Reply:
#216 ❤️